content/wiki/formatting_scheme.md

```diff
@@ -21,6 +21,21 @@ Shaders are usually licensed under the former two licenses OR the Unlicense.
 Most Linebender projects files are written in Rust.
 For those files, the formatting produced by `rustfmt` is required.
 
+The following rustfmt config is recommended:
+
+```toml
+# LINEBENDER RUSTFMT CONFIG - v1
+# Ensure lines end with \n even if the git configuration core.autocrlf is not set to true
+newline_style = "Unix"
+
+# `Foobar { foo, bar }` is more readable than `Foo { foo: foo, bar: bar }`
+use_field_init_shorthand = true
+
+# Forces let else blocks to always be their own line(s)
+single_line_let_else_max_width = 0
```

Comment:
Obviously this isn't used at all in Linebender yet. This is a reasonable value, but I'd lean more towards keeping the default without very good reason, for consistency with the rest of the ecosystem.

This is certainly more taste based than the other two options here.

Reply:
Hmm. Fair. I don't care that much either way.

Reply:
Yeah we approved it in [xilem#305](https://github.com/linebender/xilem/pull/305). What I said back then still applies for me: *it's quite minor and so I don't personally care much either way.*

I believe this was originally added by you @PoignardAzur. So if you still think it's worth having, then I'm fine with it and don't want to bikeshed it. However, if you're not arguing for its benefit, then I don't think we have anyone doing that and we should omit it by default as Daniel mentioned.

Reply:
Sorry, I dropped the ball on this for a while. I'll make a separate PR to add it with a justification.